addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -65,7 +65,7 @@ export class WebsocketWorker {
         this.lastNotificationId = 0;
         this.messageWaitQueue = [];
         this._forceUpdateChannels = debounce(this._forceUpdateChannels, 300);
-        this._updateChannels = debounce(this._updateChannels, 0);
+        this._debouncedUpdateChannels = debounce(this._updateChannels, 500);
```

Comment:
Won't this create issues in other flows as well?

Where if we receive updates on a channel before subscribing (and there is another last id from elsewhere), we're missing the updates during the debounce?

Shouldn't for now only increase the debounce when subscribing a presence specifically?

Reply:
The same issue exists with `forceUpdateChannels` but yes we can play safe and add a debounce dedicated to presences.